.github/workflows/release.yml

```diff
@@ -63,6 +59,9 @@ jobs:
 
       - uses: docker/setup-buildx-action@v2
 
+      - name: "Add Git safe.directory" # Go 1.18+ started embedding repo info in the build
+        run: git config --global --add safe.directory /__w/tyk/tyk
```

Comment:
Were I can read about this special dir? And why it should be in git on this specific step?

Reply:
- https://git-scm.com/docs/git-config/2.35.2#Documentation/git-config.txt-safedirectory
- go 1.18 started bundling build information
- here to prevent the vague dubious ownership error it reports. The underlying issue is we don't have the same chown somewhere somehow, and this makes git not care about it. Future git will have a disabled flag for this: https://marc.info/?l=git&m=164989570902912&w=2

Reply:
Question was more about `/__w/tyk/tyk` what is this file, and why git even care about it. But if it works I guess its fine.